docs/en/integrations/index.mdx

```diff
@@ -188,4 +188,5 @@ We are actively compiling this list of ClickHouse integrations below, so it's no
 |Tableau|<img src={require('./images/logos/logo_tableau.png').default} class="image" alt="Tableau logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data visualization|Interactive data visualization software focused on business intelligence|[Documentation](https://clickhouse.com/docs/en/connect-a-ui/tableau-and-clickhouse/)|
 |Visual Studio Client|<img src={require('./images/logos/logo_vs.png').default} class="image" alt="VS logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Language client|Visual studio lightweight client|[Marketplace](https://marketplace.visualstudio.com/items?itemName=fanruten.clickhouse-light)|
 |Easypanel|<img src={require('./images/logos/logo-easypanel.png').default} class="image" alt="Easypanel logo" style={{width: '3rem', 'background-color': 'transparent'}} />|Deployment method|It's a modern server control panel. You can use it to deploy ClickHouse on your own server.|[Website](https://easypanel.io), <br/>[Documentation](docs/en/integrations/deployment/easypanel/index.md)|
+|VulcanSQL|<img src={require('./images/logos/logo-vulcansql.png').default} class="image" alt="VulcanSQL logo" style={{width: '3rem', 'background-color': 'transparent'}} />|Data API Framework|It's a Data API Framework for data applications that helps data folks create and share data APIs faster. It turns your SQL templates into data APIs. No backend skills required.|[Website](https://vulcansql.com/)|
```

Comment:
SHouldn't the link navigate to the ClickHouse connector? https://vulcansql.com/docs/connect/clickhouse

Reply:
I fixed it. Thanks